src/extensions/extension-discovery.ts

```diff
@@ -260,6 +265,22 @@ export class ExtensionDiscovery {
     return extensions;
   }
 
+  /**
+   * Returns the symlinked path to the extension folder,
+   * e.g. "/Users/<username>/Library/Application Support/Lens/node_modules/@publisher/extension"
+   */
+  protected getInstalledPath(name: string) {
+    return path.join(this.nodeModulesPath, name);
+  }
+
+  /**
+   * Returns the symlinked path to the package.json,
+   * e.g. "/Users/<username>/Library/Application Support/Lens/node_modules/@publisher/extension/package.json"
+   */
+  protected getInstalledManifestPath(name: string) {
+    return path.join(this.getInstalledPath(name), manifestFilename);
+  }
```

Comment:
It is true that I don't understand the whole of our extensions install process. However, I would like to know why we have to care about symlinks at all. Why not just "require" the package from `~/.k8slens/extensions` without needing symlinks?

Reply:
Probably for making it easy to install the dependencies. Making a big change to the extensions npm stuff will be a separate task in any case.

Reply:
of course